ValidateOtp/handler.ts

```diff
@@ -128,12 +129,15 @@ const invalidateOtp = (
     .map(() => true);
 
 export function ValidateOtpHandler(
-  redisClient: RedisClient
+  redisClient: RedisClient,
+  logPrefix: string = "ValidateOtpHandler"
 ): IGetValidateOtpHandler {
-  return async (_, payload) =>
-    retrieveOtp(redisClient, payload.otp_code)
-      .mapLeft<IResponseErrorInternal | IResponseErrorNotFound>(() =>
-        ResponseErrorInternal("Cannot validate OTP Code")
+  return async (context, payload) => {
+    const otpCode = payload.otp_code;
+    const errorLogMapping = mapWithPrivacyLog(context, logPrefix, otpCode);
+    return retrieveOtp(redisClient, otpCode)
```

Comment:
The new constant seems useless, can we remove it?
```suggestion
    const errorLogMapping = mapWithPrivacyLog(
      context,
      logPrefix,
      payload.otp_code
    );
    return retrieveOtp(redisClient, payload.otp_code)
```